yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala

```diff
@@ -287,19 +330,19 @@ private object YarnClusterDriver extends Logging with Matchers {
       sc.listenerBus.waitUntilEmpty(WAIT_TIMEOUT_MILLIS)
       data should be (Set(1, 2, 3, 4))
       result = "success"
+
+      // Verify that the config archive is correctly placed in the classpath of all containers.
```

Comment:
Not changing this will cause test broken, because original calling `val configFromExecutors = sc.parallelize(1 to 4, 4)` after `sc.stop()` in finally will failed with error
```
ApplicationMaster: Final app status: FAILED, exitCode: 15, (reason: User class threw exception: java.lang.IllegalStateException: Cannot call methods on a stopped SparkContext.
```

After this patch, this problem is exposed. So I put them before `sc.stop`